src/utils/validate-handle.ts

```diff
@@ -0,0 +1,29 @@
+import { type Indexer } from '../indexer'
+
+export type ValidateHandleError = 'existed' | 'lengthInvalid' | 'charsInvalid'
+
```

Comment:
Add some helpful comments here to explain the behavior of this function so that developers are aware of what formats are accepted?